src/test/java/com/transferwise/openbanking/client/test/TestAspspDetails.java

```diff
@@ -35,6 +35,7 @@
     private boolean registrationUsesJoseContentType;
     private boolean detachedSignatureUsesDirectoryIssFormat;
     private boolean registrationRequiresLowerCaseJtiClaim;
+    private String issuer;
```

Comment:
not sure if we have a better naming convention for this stuff, for example the audience is defined as `PrivateKeyJwtAuthenticationAudience`